design/workflow.md

```diff
@@ -52,16 +53,18 @@ jobs:
     requires: ~pr
 ```
 
-### SCM Branch-specific jobs
+### Branch filtering (SCM Branch-specific jobs)
 
-Use exact match of a string (eg: `staging`) or a regex filter (eg: `/^user-.*$/`) after `~commit:` to denote a job only run when code changes are made to branches matching that pattern.
+Use exact match of a string (eg: `staging`) or a regex filter (eg: `/^user-.*$/`) after `~commit:` or `~pr:` to denote a job only run when code changes are made to branches matching that pattern. A workflow configuration is obtained from past sha of a pipeline repositroy if a pipeline restarts from past events which are triggered by branch filtering workflow.
```

Comment:
a past sha*

Please correct me if I'm misunderstanding, but I think the second part of the sentence can be reworded as:

`if an event which was was triggered by a branch filtering workflow is restarted`

Reply:
It's reasonable! fixed!